src/ert/analysis/_es_update.py

```diff
@@ -130,6 +130,9 @@ def _save_temporary_storage_to_disk(
                 target_fs.save_surface_data(
                     key, realization, surface_config.base_surface_path, matrix[:, i]
                 )
+        elif config_node.getImplementationType() == ErtImplType.FIELD:
+            for i, realization in enumerate(iens_active_index):
+                target_fs.save_field_data(key, realization, matrix[:, 1])                
```

Comment:
`target_fs.save_field_data(key, realization, matrix[:, 1])` should be
`target_fs.save_field_data(key, realization, matrix[:, i])`